packages/create-pantheon-decoupled-kit/src/templates/next-wp/lib/Menus.js

```diff
@@ -29,8 +29,44 @@ export async function getFooterMenu() {
 				'No footer menu data available. Try customizing your query.',
 			);
 		}
-		const menuItems = menu?.menuItems?.edges?.map(({ node }) => node);
-		return { menuItems, menuItemHeaders: headers };
+		const footerMenuItems = menu?.menuItems?.edges?.map(({ node }) => node);
+		return { footerMenuItems, footerMenuItemHeaders: headers };
+	} catch (error) {
+		console.error(error.message);
+		return [];
+	}
+}
+
+export async function getHeaderMenu(id) {
+	const query = gql`
+		query NavigationMenu {
+			menu(idType: NAME, id: "Example Menu") {
+				menuItems {
+					edges {
+						node {
+							id
+							path
+							label
+						}
+					}
+				}
+			}
+		}
+	`;
+
+	try {
+		const {
+			data: { menu },
+			headers,
+		} = await client.rawRequest(query);
+
+		if (!menu) {
+			throw new Error(
+				'No header menu data available. Try customizing your query.',
+			);
+		}
+		const headerMenuItems = menu?.menuItems?.edges?.map(({ node }) => node);
+		return { headerMenuItems, headerMenuItemHeaderes: headers };
```

Comment:
@CobyPear happened to notice this after merging. `headerMenuItemHeaderes` - Headeres looks like a typo.